### PR TITLE
Issue 1017 by pjohans: pass error from opensearch to searchresult

### PR DESCRIPTION
**Pull request by pjohans**
grep error from opensearch (if any)

### CODE REVIEW

**Comment by pjohans** [comment]
this one works with
https://github.com/ding2/ting_search/pull/19

File: lib/request/TingClientSearchRequest.php [hunk 1]
```diff
@@ -213,6 +213,12 @@ public function processResponse(stdClass $response) {
     $searchResult->numTotalCollections = self::getValue($searchResponse->result->collectionCount);
     $searchResult->more = (strcasecmp('true', self::getValue($searchResponse->result->more)) == 0);
 
+    $error = isset($searchResponse->error) ? self::getValue($searchResponse->error) : NULL;
+    if(isset($error)){
+      $searchResult->error = $error;
+    }
```
Comment (kasperg) [inline lib/request/TingClientSearchRequest.php]: This change makes little sense to me. Right above this we check for the same `error` property and throw an exception if it is present. Thus we should never be able to reach this part of the code.

**Comment by kasperg** [comment]
@pjohans: Review complete. There is probably something I have misunderstood but I cannot see how this can work

**Comment by pjohans** [comment]
Sorry, didn't notice that. Let's say this was never committed

**Comment by kasperg** [comment]
OK. I will close this.

**Pull request closed without merging**